contributors/design-proposals/apiserver-build-in-admission-plugins.md

```diff
@@ -0,0 +1,31 @@
+# Build some Admission Controllers into the Generic API server library
+
+**Related PR:**
+
+| Topic | Link |
+| ----- | ---- |
+| Admission Control | https://github.com/kubernetes/community/blob/master/contributors/design-proposals/admission_control.md |
+
+## Introduction
+
+At the moment admission controllers are implemented as plugins and they have to be compiled into the final binary in order to be used at a later time. To streamline this process especially for aggregated API servers we would like to build some plugins into the generic API server library.
+
+## Implementation
+
+The first step would be to move some plugins from [admission pkg](https://github.com/kubernetes/kubernetes/tree/master/plugin/pkg/admission) to the generic API server library. We can start off by moving the namespace lifcycle controller. 
```

Comment:
We should describe why we need to move some of these.  Perhaps a brief explanation like: "Some admission plugins are used to ensure consistent user expectations.  These plugins should be moved.  One example is the Namespace Lifecycle plugin which prevents users from creating resources in non-existent namespaces."